coalib/parsing/inline_comments.py

```diff
@@ -0,0 +1,30 @@
+import re
+
+
+def read_ignore_comments(original_line, start_comment, end_comment=''):
+    """
+    Read Ignore Comment(s) already present in the line and
+    accordingly return a list of bears previously ignored.
+    """
+    if start_comment not in original_line:
+        return
+
+    if end_comment is not '':
+        end_comment = ' ' + end_comment
+
+    ignore_regex = re.compile(re.escape(start_comment)
+                              + r'\s(\wgnore|\woqa)(\s\S+)+'
```

Comment:
as previously noted at https://github.com/coala/coala/pull/5069#discussion_r186591327
this should be a module level constant